### ISSUE TITLE
A few accessibility issues.

### ISSUE THREAD

**Issue by TheQuinbox** [TwidereProject/TwidereX-iOS#51]
I'm a fully blind Twitter and Mastodon user, and when I saw this app, got excited that I might not have to switch between Twitterrific and Metatext, could just use one app for both. However, there are a few accessibility problems, listed below.

1. Tweet text doesn't read. I can't confirm this with Mastodon because I didn't try it yet, but the text of tweets doesn't read with the VoiceOver screen reader. I can read the username, the username if someone's being quoted/retweeted, the time, if I click on it even the source, but not the text.
2. Some button names are unclear. This one's more minor, but for example, the reply button is labeled with something like, Arrow, turn, up, left. After pressing it I found out that it was reply, but I didn't know that without clicking on it. The rest of the buttons are okay though.
3. The tab bar isn't labeled. The tabs at the bottom of the screen have no label to VoiceOver, so I have no idea what tab I'm switching to. 
That's all the ones I've found for now, but I'll drop in if I find more.

**Comment by MainasuK** [comment]
Hi @TheQuinbox,

I'm sorry the inconvenience of accessbility feature lacking. I'll make a patch in the next build. Also, we are welcome the a11y missing reports or feature request. SO we can keep improving the a11y supports.

**Comment by TheQuinbox** [comment]
Awesome, thank you. Always good to see developers that care about A11y :)

**Comment by TheQuinbox** [comment]
Hello,
    I just installed the new update, and I love how the timeline reads! There are still a few issues though, and one feature suggestion. Once these are done though I see myself switching to use this as a primary client! NOte that these are all problems with Twitter, I have yet to test Mastodon support.

1. The tab bar is unlabeled. The tabs across the bottom of the screen have no label, at least to VOiceOver, so I have no idea what tab I'm switching to.
2. The likes and reteweets buttons don't tell you which they are. This is ore minor, but the controls on the screen that tell you how many likes and retweets a tweet has just say the number, not if it's likes or retweets. I'm also not sure why they're buttons, because clicking on them doesn't seem to do anything.
3. The icon in the top left corner is unlabeled. This seems? To take you to your profile. According to VoiceOver's image description guessing, it's an image of a person, but I didn't know it was profile until I pressed it.
4. The buttons above the tweet list in someone's profile are unlabeled. I have no idea what these 3 buttons above the tweet list are, but they're just read as "Button. Heading". Which doesn't make any kind of sense.
5. And the feature request. Add VoiceOver actions to tweets. This basically makes it so that we can swipe up or down to like, retweet, quote, view the source, etc. without having to double tap on the tweet, swipe over to the button, and press it. If you want examples of clients that do this right, the official Twitter for iPhone and Twitterrific both do it.

Thanks for looking into and fixing accessibility problems! :)

**Comment by MainasuK** [comment]
Hi @TheQuinbox,

The last build make the timeline ready for VoiceOver. The other parts still needs some efforts to make it works. But unfortunately I'm not available in recent weeks. But once I come back there should be a fix for these issues.

And thank you for your feedbacks.

**Comment by TheQuinbox** [comment]
Awesomeness, thank you. I love how this client is looking so far :)

**Comment by MainasuK** [comment]
The some accssibility issues related with post interaction resovled. I see there are still some a11y supports missing. And more improves will be added in the future build.
